server/server.go

```diff
@@ -707,6 +707,24 @@ func (j *jsonRPCHub) GetStorage(stateRoot types.Hash, addr types.Address, slot t
 	return res.Bytes(), nil
 }
 
+func (j *jsonRPCHub) Get(rootHash types.Hash) ([]byte, error) {
```

Comment:
IMO we should extend State interface with Get method, that way we will avoid snapshot getting. Just extend State interface with Get and call state.Get directly from here.